docs/Custom_Device_Types.md

```diff
@@ -102,6 +102,16 @@ The *Close* case executes after the PCL has finished executing. You should close
 
 **Note:** Because the PCL has terminated, channel values cannot be read or written in the Close case.
 
+### Inline-Async Hardware Interface
+
```

Comment:
```suggestion

By checking the **Use Inline-Async API** for the *Inline HW Interface* Custom Device Execution Mode in the niveristand-custom-device-wizard, an inline-async custom device template will be generated.

![](picture to be added from the comment snip)

By using the [Inline-Async-API](https://github.com/ni/niveristand-custom-device-development-tools/tree/main/inline-async-api) the inline-async template framework provides initializing/launching/cleanup of the asynchronous VIs, error handling/reporting, and data transfer between inline and asynchronous VIs.

```

Reply:
Please also add the below picture in the above change:

![image](https://user-images.githubusercontent.com/62251693/142393277-89f4233c-f96b-498d-a60f-d4d0543ad9cf.png)